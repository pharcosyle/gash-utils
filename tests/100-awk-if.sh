echo -e "foo\nbar" | awk -f tests/data/awk/if.awk
