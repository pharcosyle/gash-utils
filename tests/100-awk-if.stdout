1:foo
false
1:bar
false
